### PR TITLE
Remove all non essential HTML files from translation in this repo

### PR DESCRIPTION
**Pull request by teolemon**
Remove all non essential HTML files from translation in this repo.
They will be moved to  https://github.com/openfoodfacts/openfoodfacts-web/pull/4

### CODE REVIEW

**Comment by teolemon** [comment]
❌ No sources found for '/openfoodfacts-server/madenearme/madenearme-world.html' pattern. Check the source paths in your configuration file
⚠️  Skipping file 'index.html' because it is already uploading/uploaded
⚠️  Skipping file 'contribute.html' because it is already uploading/uploaded
❌ No sources found for '/lang/obf/en/texts/data.en.html' pattern. Check the source paths in your configuration file
⚠️  Skipping file 'discover.html' because it is already uploading/uploaded
⚠️  Skipping file 'faq.html' because it is already uploading/uploaded
❌ No sources found for '/lang/en/obf/texts/mobile-app-open-beauty-facts.html' pattern. Check the source paths in your configuration file
⚠️  Skipping file 'nutrition-traffic-lights.html' because it is already uploading/uploaded
⚠️  Skipping file 'index.html' because it is already uploading/uploaded
⚠️  Skipping file 'index.html' because it is already uploading/uploaded
❌ Current execution finished with errors

**Comment by teolemon** [comment]
create mode 100644 madenearme/madenearme_lol.html
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.        
remote: warning: See http://git.io/iEPt8g for more information.        
remote: warning: File 7ae4aaba58c933f311dfcc842cbab6713754bc6f is 70.93 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB        
To https://github.com/openfoodfacts/openfoodfacts-server.git
 ! [remote rejected] l10n_master_gh -> l10n_master_gh (shallow update not allowed)
error: failed to push some refs to 'https://github.com/openfoodfacts/openfoodfacts-server.git'

**Comment by teolemon** [comment]
oh shit @stephanegigandet @hangy

**Comment by hangy** [comment]
>  create mode 100644 madenearme/madenearme_lol.html
> remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.        
> remote: warning: See http://git.io/iEPt8g for more information.        
> remote: warning: File 7ae4aaba58c933f311dfcc842cbab6713754bc6f is 70.93 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB        
> To https://github.com/openfoodfacts/openfoodfacts-server.git
>  ! [remote rejected] l10n_master_gh -> l10n_master_gh (shallow update not allowed)
> error: failed to push some refs to 'https://github.com/openfoodfacts/openfoodfacts-server.git'

Well yeah, the repo is bloated with large files …

**Comment by teolemon** [comment]
@hangy @stephanegigandet I've left the strict minimum in the config file: 
"remove everything that's not strictly necessary"

**Comment by teolemon** [comment]
@hangy @stephanegigandet https://github.com/openfoodfacts/openfoodfacts-web/pull/4

File: .github/workflows/crowdin.yml [hunk 1]
```diff
@@ -77,3 +82,5 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
         CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+        project_id_env: CROWDIN_PROJECT_ID
+        api_token_env: CROWDIN_PERSONAL_TOKEN
```
Comment (teolemon) [inline .github/workflows/crowdin.yml:86]:
```suggestion
```

**Review by stephanegigandet — APPROVED** [review]

**Comment by stephanegigandet** [comment]
Moving /lang to an openfoodfacts-web repo sounds like a good idea.

**Comment by hangy** [comment]
> Moving /lang to an openfoodfacts-web repo sounds like a good idea.

I generally agree. (And as I've said before, I'd do the same with taxonomies, because of the bloat they add to the repo size.) The only issue with web is that is potential conflicts with script changes like the Foundation update, because IIRC the HTML sometimes contains specific CSS/JS.

**Pull request merged**